slurm/adam.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#SBATCH --job-name=ADAM
+#SBATCH --account=borrowed
+#SBATCH --partition=scavenge
+#SBATCH --qos=scavenge
```

Comment:
Do we want to run on `scavenge`? Running on `ephemeral` gives us a guarantee of 1 hour minimum runtime which given we're only requesting an hour for this job seems reasonable?

Reply:
I wasn't aware of the guarantee, but that makes sense.

Reply:
It's detailed here in the [shared resources policy](https://github.com/isi-vista/saga-cluster/wiki/Adding-a-Project-to-SAGA:-Policy-(Work-in-Progress)#shared-resources)